remote-trait-object/src/service.rs

```diff
@@ -17,15 +17,20 @@
 use serde::{Deserialize, Serialize};
 
 pub type InstanceId = u32;
+pub type MethodId = u32;
 
 /// This struct represents an index to a service object in port server's registry
 #[derive(PartialEq, Serialize, Deserialize, Debug, Clone, Copy)]
 pub struct ServiceObjectId {
     pub(crate) index: InstanceId,
 }
 
-pub trait Dispatch {
-    fn dispatch_and_call(&self, method: String, args: &[u8]) -> Vec<u8>;
+/// Exporter sides's interface to the service object. This will be implemented
+/// by each service trait's unique wrapper in the macro
+pub trait Dispatch: Send + Sync {
+    fn dispatch_and_call(&self, method: MethodId, args: &[u8]) -> Vec<u8>;
 }
 
-pub trait Service: Dispatch + Send + Sync {}
+pub trait Service: Send + Sync {
```

Comment:
I'm happy to see that `Dispatch` is removed from `Service`. It made me confusing before.